### PR TITLE
Remove the GetUniqueTexlRuntimeName virtual from TexlFunction

### PR DESCRIPTION
**Pull request by CarlosFigueiraMSFT**
This is only used by Power Apps, no need to be in Power Fx

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs [hunk 1]
```diff
@@ -180,6 +180,12 @@ public IService ParentService
             }
         }
 
+        // Used in tests
+        internal (string Namespace, string Name) GetName()
+        {
+            return (base.Namespace.Name.Value, base.Name);
```
Comment (lesaltzm) [inline src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs:186]: If it's just in those tests, can't you change the tests to, for example: 

 

func =>
                {
                    return
                        func.Namespace.Name.Value == "LQA" &&
                        func.Name == "GetAnswersFromText";
                }

**Review by lesaltzm — APPROVED** [review]

File: src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs (hunk 1 above)
Comment (CarlosFigueiraMSFT) [reply src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs]: Updated it to add internals visibility to the connectors test project

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**